tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py35-django22-drf{39,latest},py38-django{22,30}-drf{39,latest},quality, docs
+envlist = py35-django22-drf{39,latest},py38-django{22}-drf{39,latest},py38-django{30,31}-drf{39,latest},quality, docs
```

Comment:
Was there a reason the Django versions needed to be split apart in separate entries like this?

Reply:
it should be `py38-django{22}-drf{39,latest},py38-django{30,31}-drf{latest}` I updated the `ci.yml` but forgot this, `djangorestframework` version 3.9 supports Django only up to 2.2 so we we've to exclude DRF 3.9 environment for Django 3.0 and 3.1